Makefile

```diff
@@ -32,8 +32,9 @@ endif
 # These variables describe the locations of various files and
 # directories in the source tree.
 BASICS_DIR = $(CURDIR)/basics
-BASICS_INC = $(BASICS_DIR)/include
-BASICS_CRT_SOURCES = $(wildcard $(BASICS_DIR)/crt/*.c)
+BASICS_HEADERS_PUBLIC = $(BASICS_DIR)/headers/public
+BASICS_HEADERS_PRIVATE = $(BASICS_DIR)/headers/private
```

Comment:
This header file rename seems logically separate no?   Why not continue to using `include` ?  `include/public` +  `include/private`?